### ISSUE TITLE
Do you have your training log available?

### ISSUE THREAD

**Issue by kai-lan** [haimsaw/LoRA-MDM#10]
I am trying to train with LoRA-MDM. Somehow, the loss is increasing as I finetune with LoRA. I selected 100 styles. Is it normal that the loss goes up? What is the loss curve when you trained your model?
